nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -777,10 +777,10 @@ def exec(self, layer_name=None):
                 if conns1 != conns2:
                     different_conns_list.append(f'Allowed connections from {peer1} to {peer2}'
                                                 f' are different\n' + conns1.print_diff(conns2, self.name1, self.name2))
+                    if not self.output_config.FullExplanation:
+                        break
```

Comment:
Note this is a nested loop, and in python the `break` keyword can only break out of the inner-most loop. But in this case we need to break from both loops.